### PR TITLE
Function is used to check the netmask value and convert it into shortform (/mask) of netmask values

### PR DESCRIPTION
**Pull request by TasmiyaNalatwad**
Function is used to check the netmask value and convert it into shortform (/mask) of netmask values
Example : 255.255.255.0 = /24
255.255.252.0 = /22

Signed-off-by: Tasmiya Nalatwad <tasmiya@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by PraveenPenguin** [comment]
@TasmiyaNalatwad can you please address Travis error

File: avocado/utils/network/interfaces.py [hunk 1]
```diff
@@ -595,3 +595,14 @@ def are_packets_lost(self, peer_ip, options=None, sudo=False):
...
             raise NWException(msg)
+
+    def netmask_to_cidr(self, netmask):
+        """ Function is used to check the netmask value and convert
+        it into short form (/mask) of netmask values
+        Example : 255.255.255.0 = /24
+        255.255.252.0 = /22
+
+        :param netmask: Netmask value example 255.255.255.0
+        :return : Returns /mask value of given netmask
```
Comment (abdhaleegit) [inline avocado/utils/network/interfaces.py:606]: Why return /  , I would suggest return only integer value

Comment (abdhaleegit) [reply avocado/utils/network/interfaces.py:606]: Append / in test case

Comment (abdhaleegit) [reply avocado/utils/network/interfaces.py:606]: Add return type in doc string

File: avocado/utils/network/interfaces.py [hunk 2]
```diff
@@ -595,3 +595,14 @@ def are_packets_lost(self, peer_ip, options=None, sudo=False):
...
+        """
+        return(sum([bin(int(bits)).count("1") for bits in netmask.split(".")]))
```
Comment (abdhaleegit) [inline avocado/utils/network/interfaces.py:608]: Reformat return, space between return and operation

**Review by abdhaleegit — CHANGES_REQUESTED** [review]

File: avocado/utils/network/interfaces.py [hunk 3]
```diff
@@ -595,3 +595,14 @@ def are_packets_lost(self, peer_ip, options=None, sudo=False):
         except Exception as ex:
             msg = f"Failed to ping. {ex}"
             raise NWException(msg)
+
+    def netmask_to_cidr(self, netmask):
+        """ Function is used to check the netmask value and convert
```
Comment (abdhaleegit) [inline avocado/utils/network/interfaces.py:600]: insert a empty line after, the reason why travis is failing.. refer to other methods doc string inthis file

**Review by abdhaleegit — CHANGES_REQUESTED** [review]

**Review by abdhaleegit — APPROVED** [review]
LGTM

**Review by PraveenPenguin — APPROVED** [review]

**Pull request merged**